demos/tutorial/seed_and_extend/example2.cpp

```diff
@@ -22,7 +22,7 @@ int main()
                           endPositionV(seed)) << "\n";
 
     // Perform match extension.
-    extendSeed(seed, seqH, seqV, EXTEND_LEFT, MatchExtend());
+    extendSeed(seed, seqH, seqV, EXTEND_BOTH, MatchExtend());
     // Print the resulting seed.
```

Comment:
Because the tutorial states: "In the most simple case we simply look for matching characters in both sequences to the left and right end of the seed."
http://seqan.readthedocs.org/en/master/Tutorial/SeedAndExtend.html (Seed Extension)
I just wanted to make it consistent. Do you want me to change it back?